coalib/output/ConsoleInteraction.py

```diff
@@ -278,15 +278,26 @@ def print_diffs_info(diffs, printer):
             color='green')
 
 
+_warn_deprecated_format_str = True  # Remove when format_str is deprecated
```

Comment:
nope; I ack'd, as this isnt the bug to start using `warnings`.  It works; lets get it done.